src/js/components/TaskView.js

```diff
@@ -160,23 +160,24 @@ class TaskView extends mixin(SaveStateMixin) {
         <FilterHeadline
           inverseStyle={inverseStyle}
           onReset={this.resetFilter}
-          name="Tasks"
+          name="Task"
```

Comment:
We should probably pluralize this too, based off the total number of tasks (and not the filtered number of tasks)